app/views/users/import.haml

```diff
@@ -0,0 +1,7 @@
+%h1 Import User Data
+
+= form_with url: '#', local: true, multipart: true, authenticity_token: true do |form|
```

Comment:
Why is everything after the first apostrophe grayed out? I don't think it should be.